dataland-frontend/tests/e2e/fixtures/common/DataPointFixtures.ts

```diff
@@ -108,11 +76,11 @@ export function generateDatapoint<T, Y>(value: T | null, reports: ReferencedDocu
  * @param qualityBucket the quality bucket of the datapoint
  * @returns the generated data source and comment of the datapoint
  */
-function createQualityAndDataSourceAndComment(
+function generateQualityAndDataSourceAndComment(
   reports: ReferencedDocuments,
   qualityBucket: QualityOptions,
 ): { dataSource: CompanyReportReference | undefined; comment: string | undefined } {
-  let dataSource: CompanyReportReference | undefined = undefined;
+  let dataSource: CompanyReportReference | undefined;
   let comment: string | undefined = undefined;
```

Comment:
`= undefined` is unnecessary since the variable is undefined per default if no value assigned to it

Reply:
changed